src/routes/search.js

```diff
@@ -83,12 +97,13 @@ class SearchRoute extends Component {
   getResults() {
     let { resolver } = this.props;
     let { searchType, params } = this.state;
-    let { offset = 0, ...queryParams } = params;
+    let searchParams = this.transformQueryParams(params);
+    let { offset = 0, ...pagedParams } = searchParams;
```

Comment:
`offset` comes from the URL params, so it shouldn't come from the tranformed search params. You could leave the old destructuring and just transform `queryParams`

```
let { offset = 0, ...queryParams } = params;
let searchParams = this.transformQueryParams(queryParams);
```